### DISCUSSION TITLE
Open Toolbar in Modal Window

### DISCUSSION THREAD

**Discussion by Glint-Eye** [chrisgurney/obsidian-note-toolbar#119] [Ideas]
Note Toolbar is a great Tool for me to manage the plethora of avaible commands in reasonable groups with the commands i use on a regular basis (e.g. Group for formatting commands, Group for file commands etc).

It would be great if all of these command groups and their commands could be tugged away in a modal prompt to save on screen real estate and quickly filter groups and commands via text input.

**Comment by chrisgurney** [comment]
@Glint-Eye This is a great idea, and I can think of at least two other use cases where this UI would be super useful. Thank you!

**Comment by chrisgurney** [comment]
@Glint-Eye Check out [1.12-beta-1](https://github.com/chrisgurney/obsidian-note-toolbar/releases/tag/1.12-beta-1) for a variation of this idea. (Items are not grouped by toolbar, but all are searchable.)

[Install via BRAT](https://github.com/chrisgurney/obsidian-note-toolbar?tab=readme-ov-file#installation) and let me know what you think! _Thanks again for the idea!_

**Comment by Glint-Eye** [comment]
Tried out the new Quick tools. Its a really great addition. Especially the fact that i can also access all other toolbars not currently active for the current file. One thing i noticed though is that all the items seem to be sorted alphabetically. It would be great if the items in quick tools could respect the custom sort order as defined in the toolbar edit.

Thx again for this great addition.

**Reply by chrisgurney** [reply]
@Glint-Eye Thanks for the feedback!

I'd be open to making items in the list non-alphabetical, when using the Quick Toolbars command (i.e., the one that shows toolbars first).

Would that work for you?

**Reply by chrisgurney** [reply]
@Glint-Eye I've changed the behavior in the next release such that if you're using the "Quick Toolbars" command, on selecting a toolbar, the items will be shown in toolbar order.

Thanks for the suggestion!
